views/admin/category/allcategories.php

```diff
@@ -0,0 +1,40 @@
+<?php
+     $PAGE_TITLE="All Categories";
+     $PAGE_STYLESHEETS = "";
+     $PAGE_SCRIPTS = "";
+     require_once  "../../../templates/header.php"; 
+     require_once "../../../database_connection.php";
+?>
+<div class="container">
+    <h1 class="text-center my-3">All Categories</h1>
+    <a href="#" class="btn btn-success mb-4 offset-2">Add New Category</a>
+    <?php
+            try {
+                $stm=$conn->prepare("SELECT * FROM category");
+                $stm->execute();
+                $categories=$stm->fetchAll(); 
+              if(count($categories)>0){
+                echo "<div class='row'><div class='col-md-8 offset-2'>";
+                    echo "<table class='table'>
+                    <tr class='bg-dark text-light'>
+                        <th scope='col'>ID</th>
+                        <th scope='col'>Category Name</th>
+                        <th scope='col'>Action</th>
+                    </tr>";
+            foreach($categories as $category){
+                echo "<tr id='".$category['category_id']."'><th scope='row'>".$category['category_id']."</th>".              
+                "<td> ".$category['name']."</td>".
```

Comment:
You must not show data that was entered by the user to the screen before sanitizing it.